modules/python/src2/cv2.cpp

```diff
@@ -34,15 +34,31 @@
 #include "pycompat.hpp"
 #include <map>
 
+class ArgInfo
+{
+public:
+    const char * name;
+    bool outputarg;
+    // more fields may be added if necessary
+
+    ArgInfo(const char * name_, bool outputarg_)
+        : name(name_)
+        , outputarg(outputarg_) {}
+
+private:
+    ArgInfo(const ArgInfo&); // = delete
+    ArgInfo& operator=(const ArgInfo&); // = delete
```

Comment:
Why don't you delete these publicly and explicitly?

Reply:
"= delete" requires C++11 (OpenCV 3.4 still using C++98)

Reply:
Sorry, didn't see that. LGTM, then :+1: